src/main/java/com/github/jasminb/jsonapi/exceptions/UnregisteredClassForType.java

```diff
@@ -0,0 +1,18 @@
+package com.github.jasminb.jsonapi.exceptions;
+
+/**
```

Comment:
Hi, Thanks for your feedback. I have added the javadoc and renamed the exception.